subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -242,3 +242,16 @@ These members will be removed in Gradle 9.0.
 The enum constant `JvmVendorSpec.IBM_SEMERU` is now deprecated and will be removed in Gradle 9.0.
 
 Please replace it by its equivalent `JvmVendorSpec.IBM` to avoid warnings and potential errors in the next major version release.
+
+=== Changes in the IDE integration
+
+[[kotlin_dsl_plugins_catalogs_workaround]]
+==== Workaround for false positive errors shown in Kotlin DSL `plugins {}` block using version catalog is not needed anymore
+
+Version catalog accessors for plugin aliases in the `plugins {}` block aren't shown as errors in IntelliJ IDEA and Android Studio Kotlin script editor anymore.
+
+If you were using the `@Suppress("DSL_SCOPE_VIOLATION")` annotation as a workaround, you can now remove it.
```

Comment:
does clear IDE caches need mentioning or was that suggestion more of a precaution?

Reply:
Good point. Yes according to my tests this is required if you upgrade Gradle on a build already imported in the IDE. I added some words in this regard.